### ISSUE TITLE
Add support for modules

### ISSUE THREAD

**Issue by intel352** [heroku/heroku-buildpack-go#249]
Please add support for https://github.com/golang/vgo as an alternative to dep, govendor, etc. Vgo is the most likely candidate to be directly integrated into Go by v1.11.
Read more about it here: https://research.swtch.com/vgo

**Comment by kr** [comment]
A lot has happened with Go modules since this issue was opened. The modules proposal (for which vgo was a prototype) has been accepted, and it's been implemented in the `go` tool, and the Go team has released a beta (go1.11beta2) exposing this behavior.

Now that we have module support in an official release, it would be really useful for this buildpack to read `go.mod` so we can test it on Heroku.

**Comment by kr** [comment]
(Someone with permissions might want to rename this issue "Add support for modules" to reflect the official name for the feature.)

**Comment by freeformz** [comment]
Master now has some basic module support. I'll be releasing after #263 is merged, but if you want to try it out now by telling Heroku you want to the use unreleased version from github via:
```console
heroku buildpacks:set https://github.com/heroku/heroku-buildpack-go.git -a myapp
```

Please use the github issue tracker to report any problems as a separate issue. Heroku Support may not be able to help you since this is considered experimental. 

Please include the build output and your application name in any issue you raise. You will also need to push the branch that failed  to heroku as a branch named "moduleSupport" for additional debugging like so:

```console
git push heroku <your local branch name>:moduleSupport
```

**Comment by freeformz** [comment]
v91 is released so please try modules with `heroku/go`. I'm going to close this issue as I feel that opening issues for specific gaps/problems would be more appropriate. Thanks!
